EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc

```diff
@@ -29,7 +24,10 @@
 #include "EventFilter/RPCRawToDigi/interface/RPCTwinMuxRecord.h"
 
 RPCTwinMuxDigiToRaw::RPCTwinMuxDigiToRaw(edm::ParameterSet const& config)
-    : bx_min_(config.getParameter<int>("bxMin")),
+    : es_tm_link_map_br_token_(esConsumes<RPCAMCLinkMap, RPCTwinMuxLinkMapRcd>()),
```

Comment:
This needs the 3rd template argument denoting this will be gotten in the BeginRun